src/accounts/views.py

```diff
@@ -16,6 +18,7 @@ def register(request):
             if user.user_type == "service_provider":
                 return redirect("service_provider_dashboard")
             else:
+                ServiceSeeker.objects.create(user=user)
```

Comment:
I think the ServiceSeeker and user(user_type=="seeker") might introduced redundancy

Reply:
if a refactor is done to merge the profileapp into the accounts app (and maybe rename it to user_mgmt or something along those lines), keeping a separate ServiceSeeker model makes sense. will resolve this comment based on further review